packages/suite/src/components/suite/QrCode.tsx

```diff
@@ -66,9 +66,7 @@ export const QrCode = ({ value, className, bgColor, fgColor, showMessage }: QrCo
                 <SpanToResetWhitespace>
                     <Translation id="TR_QR_RECEIVE_ADDRESS_CONFIRM" />
                 </SpanToResetWhitespace>
-                <StyledTooltip
-                    content={<Translation id="TR_QR_RECEIVE_ADDRESS_CONFIRM_EXPLANATION" />}
```

Comment:
The key for `TR_QR_RECEIVE_ADDRESS_CONFIRM_EXPLANATION` shall be removed from `messages.ts` I don't see it used anywhere else.

Reply:
I had to force push it due to some git issues but it should be fixed